templates/Dockerfile

```diff
@@ -19,40 +19,44 @@ RUN dpkg --add-architecture i386 && \
     echo "deb mirror://mirrors.ubuntu.com/mirrors.txt bionic-backports main restricted universe multiverse" >> /etc/apt/sources.list && \
     apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -yq software-properties-common libstdc++6:i386 zlib1g:i386 libncurses5:i386 \
-                        locales ca-certificates apt-transport-https curl unzip redir iproute2 \
-                        openjdk-8-jdk xvfb x11vnc fluxbox nano libpulse0 telnet expect\
-                        --no-install-recommends && \
+        locales ca-certificates apt-transport-https curl unzip redir iproute2 \
+        openjdk-8-jdk xvfb x11vnc fluxbox nano telnet expect \
+        libc6 libdbus-1-3 libfontconfig1 libgcc1 \
+        libpulse0 libtinfo5 libx11-6 libxcb1 libxdamage1 \
+        libnss3 libxcomposite1 libxcursor1 libxi6 \
+        libxext6 libxfixes3 zlib1g libgl1 pulseaudio socat \
+        --no-install-recommends && \
     locale-gen en_US.UTF-8 && \
     # Install Android SDK
     curl -L $LINK_ANDROID_SDK > /tmp/android-sdk-linux.zip && \
     unzip -q /tmp/android-sdk-linux.zip -d /opt/android-sdk-linux/ && \
     rm /tmp/android-sdk-linux.zip && \
     # Customized steps per specific platform
     yes | sdkmanager --no_https --licenses && \
-    yes | sdkmanager emulator tools platform-tools "platforms;{{ platform }}" "system-images;{{ platform }};google_apis;x86" --verbose | uniq && \
-    echo no | avdmanager create avd -n "x86" --package "system-images;{{ platform }};google_apis;x86" --tag google_apis && \
+    yes | sdkmanager emulator tools platform-tools "platforms;{{ platform }}" "system-images;{{ platform }};google_apis;x86_64" --verbose | uniq && \
+    echo no | avdmanager create avd -n "Pixel2" --package "system-images;{{ platform }};google_apis;x86_64" --tag google_apis && \
```

Comment:
Changing the name of the emulator will break other setups if they do some communication with the avd by name

Reply:
Yes, you are right. Thanks